_posts/2020-07-09-stephan-druskat.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Language shapes reality"
+author: "@pweschmidt"
+rse: "Stephan Druskat"
+phenotype: sdruskat-rse-phenotype.png
+excerpt: "Meet Stephan Druskat, RSE at the Alexander von Humboldt University in Berlin, Germany. Listen to Stephan explain his exciting work for research in linguistics and his contributions to making software citable."
+date: 2020-07-09 8:30:00
+external_media: https://us-rse.org/rse-stories-episodes-1/2020/rse-stories-stephan-druskat-episode-24.mp3
+length: 19032316
+duration: "00:39:39"
+explicit: "no"
+resources:
+ - name: Stephan Druskat
+ - url: https://sdruskat.net
+ - name: Stephan Druskat Twitter
+   url:  https://twitter.com/stdruskat
+ - name: Alexander von Humboldt University
+   url: https://www.informatik.hu-berlin.de/en/standardseite-en?set_language=en
+ - name: MelaTAMP project (linguistics)
+   url: https://www.projekte.hu-berlin.de/en/melatamp
+ - name: RSE DE in Digital Humanities
+   url: https://dh-rse.github.io
+---
+
+Today we go to Berlin in Germany to meet with Stephan Druskat (@stdruskat). Stephan is a research software engineer at
```

Comment:
```suggestion
Today we go to Berlin in Germany to meet with Stephan Druskat ([@sdruskat](https://github.com/sdruskat)). Stephan is a research software engineer at
```

Reply:
We have his Twitter in the resources, so an @ (at) in the context of GitHub would  be expected to be his GitHub alias I think!